CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# How to contribute #
+
+All help is appreciated. All contributors are unpaid volunteers. If you really want to see something added to Exiv2 - volunteer!
+By participating in this project, you agree to abide by the thoughtbot [code of conduct.](https://thoughtbot.com/open-source-code-of-conduct)
```

Comment:
Do we really need this? imho this can be summarized by "don't be an asshole and don't discriminate". And I wouldn't link to an external document that we have no control of, since thoughbot could change that document without notice to our disadvantage.

Reply:
Right. I just saw this in some of the examples that github recommends to read when writing the CONTRIBUTING document and I thought it was interesting. I will remove that line to be more concise.

Reply:
Yeah I know, github recommends that, however I think most of the stuff in there is covered by their terms of service anyway. Also, we can hope that people will be decent unless we're proven wrong.